src/middlewared/middlewared/plugins/failover.py

```diff
@@ -359,6 +357,10 @@ def sync_to_peer(self, options):
         `reboot` as true will reboot the other controller after syncing.
         """
         standby = ' standby controller.'
+
+        self.logger.debug('Pulling system dataset UUID from' + standby)
```

Comment:
This seems incorrect? Does this work with `+ standby`? Or should it be `UUID from %s', standby)`

Reply:
You wrote all the remaining code of this function using this style, I just copied it. It works perfectly.